integration/README.md

```diff
@@ -70,7 +80,10 @@ Our automated CI uses Dockerfiles to build a kernel and QEMU and set these
 environment variables. You can see the Dockerfile and the config file used to
 build the kernel for each supported architecture [here](/.circleci/images).
 
-2. Run the tests with:
+2. **Run Tests**
+
+Recall that there are 2 subdirectories with tests, generic-tests/ and gotests/.
```

Comment:
Could you also not run `go test ./...` to test both directories?

Reply:
Yes, it seems to be a bit slower, so updated it accordingly